src/Container/ErrorHandlerFactory.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * @see       https://github.com/zendframework/zend-expressive for the canonical source repository
+ * @copyright Copyright (c) 2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   https://github.com/zendframework/zend-expressive/blob/master/LICENSE.md New BSD License
+ */
+
+namespace Zend\Expressive\Container;
+
+use Interop\Container\ContainerInterface;
+use Zend\Diactoros\Response;
+use Zend\Expressive\Middleware\ErrorResponseGenerator;
+use Zend\Stratigility\Middleware\ErrorHandler;
+
+class ErrorHandlerFactory
+{
+    /**
+     * @param ContainerInterface $container
+     * @return ErrorHandler
+     */
+    public function __invoke(ContainerInterface $container)
+    {
+        $generator = $container->has(ErrorResponseGenerator::class)
+            ? $container->get(ErrorResponseGenerator::class)
+            : null;
+
+        return new ErrorHandler(new Response(), $generator);
```

Comment:
@weierophinney any chance to make it possible to configure `ErrorHandler` listeners via config? Or should I override this factory to attach my custom listeners?